infrastructure/modules/onpremise-storage/activemq/configmaps.tf

```diff
@@ -259,11 +259,13 @@ EOF
     </bean>
 
    <!-- Allows accessing the server log -->
+
+    <!--
     <bean id="logQuery" class="io.fabric8.insight.log.log4j.Log4jLogQuery"
           lazy-init="false" scope="singleton"
           init-method="start" destroy-method="stop">
     </bean>
-
+    -->
```

Comment:
Are we sure this is safe to remove? If so, I would prefer to remove it entirely instead of merely commenting it.

Reply:
Done !